DTS-STN/Alpha-Site
# Translate filter labels

# Description

[Translate filter labels](https://trello.com/c/K5HYXYI4)

The filter labels on the projects page now shows properly in english and french.
![image](https://user-images.githubusercontent.com/34345435/127174649-5b97e065-283c-4519-826d-3d3b7a12d9dc.png)
![image](https://user-images.githubusercontent.com/34345435/127174718-ee059354-8f6d-4e2c-8084-0aea5f8468c8.png)

## Acceptance Criteria

The filter labels show in english and french.


## Test Instructions

1. Navigate to the projects page
2. Notice that the filter labels are in the language you've chosen for the page and not just english

## Help Requested

I am unsure of how to use the i18n package to get translations inside of `getStaticProps` since it's only used as a react hook currrently, so I've done something kind of ugly
![image](https://user-images.githubusercontent.com/34345435/127175029-20cc3718-9add-4d00-b5ad-54de9a3e5071.png)

If anyone can point me in the right direction to do this properly, I would appreciate the help. I did consider moving the translation into the Filter component and I may still end up doing it that way.

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
